source/Weapon.cpp

```diff
@@ -155,6 +156,11 @@ void Weapon::LoadWeapon(const DataNode &node)
 				damage[SHIELD_DAMAGE] = value;
 			else if(key == "hull damage")
 				damage[HULL_DAMAGE] = value;
+			else if(key == "disabled damage")
+			{
+				damage[DISABLED_DAMAGE] = value;
+				disabledDamageSet = true;
+			}
```

Comment:
@Zitchas @LocalGod79 
`"disabled damage"` is the value of the hull damage done when the target disabled, not a percentage of the standard hull damage that gets done when the target is disabled. Not setting a disabled damage value means that disabled damage defaults to being the same as the hull damage, which is the same behavior as right now. But because of `disabledDamageSet` seen here, one is able to specify that a weapon has a disabled damage of 0 and it will actually work, unlike most other attributes where a value of 0 is the same as if you never mentioned it (which is what you're assuming to be the case). (e.g. `"jump fuel" 0` causes hyperdrives/jump drives to consume the normal 100 or 200 fuel since no such boolean exists to identify if the 0 input is suppose to actually be used or not.)

Reply:
Thanks for the explanation. 

So the default `disabled damage` is "whatever the hull damage is", not 0. That sounds good to me.